academic-services/knowledge-exploration-service/tutorial-entity-linking.md

```diff
@@ -68,11 +68,11 @@ The next step in entity linking is determining an appropriate confidence score t
 The interpretation log probability value actually represents the sum of two *different* log probabilities which we need to disentangle. The first log probability, grammar path log probability, is associated with the different attributes matched in the natural language grammar that the Interpret API uses, and is used to represent how likely an interpretation is to be correct based on rules defined in the grammar. The second log probability, [entity log probability](how-to-index-data.md#entity-log-probability), is associated with the top matching entity for the interpretation.
```

Comment:
>The first log probability, grammar path log probability, is associated with the different attributes matched in the natural language grammar that the Interpret API uses [](start = 136, length = 168)

The first is the log probability associated with penalties applied in the grammar, normally attached to [attribute matching rules](reference-grammar-syntax.md#item-element) in the natural language grammar used by the Interpret API